firebase.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need this file?

Reply:
After your tools PR it won't be needed, this was added so I could do `firebase use` for the project requirement - which requires a valid Firebase project directory.

Will remove, thanks